README.md

```diff
@@ -1,79 +1,98 @@
-  ```
-  {
-    "ethereum": 1,
-    "wrapped-bitcoin": 60,
-    "maker": 5,
-    "matic-network": 6,
-    "idle": 79,
-    "richochet": 77
-  }
-  ```
-  :warning: `richochet` is not a typo, that's the Coingecko ID for RIC token price
-* Create an `HTTP` **for each** of the public addresses you used in the previous step:
-  * Set the name this connection as the public address
-  * Set the `Login` to the public address
-  * Set the `Password` to the private key for the public address
+```
 
 ## Optional
-* Navigate to `Admin > Variables` and add the following to change dag schedule:
-  * `distribution-schedule-interval` - Dag `distribute` (Default - `0 * * * *`)
-  * `harvester-schedule-interval` - Dag `harvester` (Default - `0 * * * *`)
-  * `watch-schedule-interval` - Watch stream dag (Default - `50 * * * *`)
-  * `tellor-schedule-interval` - Reporting to Tellor (Default - `*/5 * * * *`)
-  * `swap-schedule-interval` - Swap RIC stream to matic (Default - `0 * * * *`)
-  * `block-poll-schedule-interval` - Block poll (Default - `*/15 * * * *`)
-  * `close-schedule-interval` - Close streams (Default - `None`)
-  * `max-gas-price` - To set the max gas price (Default - `33`)
+* If needed you can change variables or connections values from GUI
+* Navigate to `Admin > Variables`  or `ADMIN > Connctions ` and change your values
 
-# Run
-Run the keeper using Docker Compose
+# If you have a aws account, you can deploy your keeper using terraform (this is the recomended way, all firewall rules are already set up)
+
+Rename terraform template file to terraform.tfvars
 ```
-echo -e "AIRFLOW_UID=$(id -u)" > .env
-docker-compose up
+cp terraform/terraform.{tmpl.tfvars,tfvars}
 ```
-Airflow runs on port 80 so navigate to http://localhost to access the UI. Once things have booted up, log in with username `airflow` and password  `airflow`.
 
-## Run as daemon
-Use:
+Fill  all variables contained in terraform.tfvars
+you can customize schedule values by editing .vars.tmpl file 
+```
+keeper_repository = ""                  # repository used for deployement
+keeper_repository_branch = ""           # branch to deploy - defaults to master
+key1 = ""                               # address1 key
+key2 = ""                               # address2 key
+key3 = ""                               # address3 key
+key4 = ""                               # address4 key
+key5 = ""                               # address5 key
+key6 = ""                               # address6 key
+key7 = ""                               # address7 key
+address1 = ""                           # address1
+address2 = ""                           # address2
+address3 = ""                           # address3
+address4 = ""                           # address4
+address5 = ""                           # address5
+address6 = ""                           # address6
+address7 = ""                           # address7
```

Comment:
Let's map these to the names of what they are in the GUI

Reply:
done